src/theme/theme.ts

```diff
@@ -41,7 +41,7 @@ const globalStyles = css.global`
     --top-project-background-color: ${topProjectBackgroundColor};
     --non-donatable-project-background-color: ${nonDonatableProjectBackgroundColor};
     --primary-color-new: ${primaryColorNew};
-    --explore-range-background-new: ${exploreRangeBackgroundColorNew};
+    --explore-range-background-new: ${exploreRangeSliderColorNew};
```

Comment:
Why is this done? If this corresponds to `exploreRangeBackgroundColorNew` variable, since that needs to be removed, remove this as well. Also try to keep the key and value name similar to reduce confusions in the code.